src/highlighting.c

```diff
@@ -577,7 +577,12 @@ static void styleset_common_init(GKeyFile *config, GKeyFile *config_home)
 	get_keyfile_int(config, config_home, "styling", "line_height",
 		0, 0, &common_style_set.styling[GCS_LINE_HEIGHT]);
 
+	if (&common_style_set.wordchars)
+		g_free(common_style_set.wordchars);
 	get_keyfile_wordchars(config, config_home, &common_style_set.wordchars);
```

Comment:
looks like the same fix would be required in `highlighting_init_styles()` (around line 1059), wouldn't it?